RedHatInsights/vulnerability-ui
# fix(SystemCVEs): RHINENG-11464 inventory tab advisory filter

Fix default advisory filter in vulnerability in inventory system detail

How to test
1. Go to Inventory and open a system
2. Go to Vulnerability tab
3. Observe Advisory Available filter by default

Before

![image](https://github.com/user-attachments/assets/ede64e66-ec33-4511-bb52-154ca400a74c)

After

![image](https://github.com/user-attachments/assets/30196808-a497-42e4-8c60-648ff7335139)
